components/script/parser.out

```diff
@@ -0,0 +1,8079 @@
+Created by PLY version 3.3 (http://www.dabeaz.com/ply)
```

Comment:
This looks like an accident.

Reply:
Thanks for catching that, yes, it was an accident. I've fixed it.